src/transformers/models/groupvit/configuration_groupvit.py

```diff
@@ -343,3 +350,44 @@ def to_dict(self):
         output["vision_config"] = self.vision_config.to_dict()
         output["model_type"] = self.__class__.model_type
         return output
+
+
+class GroupViTOnnxConfig(OnnxConfig):
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("input_ids", {0: "batch", 1: "sequence"}),
+                ("pixel_values", {0: "batch"}),
```

Comment:
Same comment as in the OWLViT PR: should this be

```suggestion
                ("pixel_values", {0: "batch", 1: "num_channels", 2: "height", 3: "width"} }),
```